### DISCUSSION TITLE
South American faction

### DISCUSSION THREAD

**Discussion by CLAFMB** [Vdauphin/HeartsAndMinds#1620] [Question and answer]
If it is possible to add a South American faction such as Venezuelan or Colombian military forces, it would be very good, considering that the hearts and minds mode is widely used by Brazilian clans

**Comment by Vdauphin** [comment]
Hello,

Feel free to share mod link here https://github.com/Vdauphin/HeartsAndMinds/issues/738
I will add them when I will update the faction list

Cheers

**Comment by Oldman-BR** [comment]
Fala aí meu amigo. Eu fiz essa mudança. Se precisar de uma ajuda me chama e te ajudo. Abraço.
